src/components/Application.tsx

```diff
@@ -450,11 +450,17 @@ export class Application extends React.Component<Props, State> {
   }
 
   private handleClearBbox() {
+    // Only deselect if the selected feature is not a job.
+    let selectedFeature = this.state.selectedFeature
+    if (selectedFeature && selectedFeature.properties.type !== TYPE_JOB) {
```

Comment:
This check is done in 3 places in the Application.tsx. Can we move this to a private function? 

This may be helpful down the line when we have more types than just Job or Scene. In the future, we may add things like Product Lines which would make this conditional grow quite a bit more.

Reply:
Ha, I actually had it in a private function and then broke it out because it seemed like an oddly specific function. But I think you're right that it shouldn't be repeated. I'll think of a way to structure as a function in a way that's as clear as possible.